packages/volto-slate/news/5926.bugfix

```diff
@@ -0,0 +1 @@
+Fix insert/remove element edgecase bug in slate. @razvanMiu
```

Comment:
What edge case does this fix? Please describe the edge case, and how does what you did fix it.